Gemfile

```diff
@@ -62,6 +62,7 @@ gem 'typhoeus'
 gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
 gem 'virtus'
 gem 'will_paginate'
+gem 'zero_downtime_migrations'
```

Comment:
Wasn't this added in a prior commit - why did it need to be removed?

Reply:
i removed it because we ran into a bug in the deploy for edu claims and i needed to write a patch